tests/integration/cmd_app_test.go

```diff
@@ -0,0 +1,63 @@
+package integration
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var _ = Describe("odoCmdAppE2e", func() {
+	var project string
+	var context string
+
+	appName := "app"
+	cmpName := "nodejs"
```

Comment:
You can directly use the value instead of assigning it to a variable and then use it in the script.